docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
+    db:
+        image: docker.io/library/postgres:16-alpine
```

Comment:
More of a question than a review comment, but since I don't think we plan to do transaction enqueueing, I assume we'll run our own RDS instance for this service? In that case we can actually run Postgres 16.

Reply:
I opted for the latest version arbitrarily. Regardless of the service providing us with Postgres, the docker-compose was more intended for development, eventually I wanted to add an echo server for sending requests to. I don't have an opinion regarding RDS or other Postgres hosting solutions. As long as they are quick to spin up, I think we don't need much else.

Reply:
Makes sense, I only brought it up because I feel production and dev Postgres versions should match when we're getting into nitty gritty feature specifics like we will in this repo. 🤷

Reply:
I'll bring up the conversation, thank you for raising this point @bretthoerner